src/main.cpp

```diff
@@ -679,6 +681,12 @@ void setup()
     digitalWrite(SX126X_ANT_SW, 1);
 #endif
 
+#ifdef CANARYONE
+    // This is required to give the GPS time to start up as this board has a separate power supply 
```

Comment:
Can you define something more generic? Other boards may benefit from this delay too, and it's tied in with the `PIN_PWR_EN` so you can name it appropriately. (perhaps `PIN_PWR_DELAY_MS`?)